src/main/kotlin/com/dansplugins/factionsystem/command/faction/help/MfFactionHelpCommand.kt

```diff
@@ -0,0 +1,36 @@
+package com.dansplugins.factionsystem.command.faction.help
+
+import com.dansplugins.factionsystem.MedievalFactions
+import com.dansplugins.factionsystem.faction.MfFactionMember
+import com.dansplugins.factionsystem.player.MfPlayer
+import dev.forkhandles.result4k.onFailure
+import org.bukkit.ChatColor.RED
+import org.bukkit.ChatColor.AQUA
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+import java.util.logging.Level
+
+class MfFactionHelpCommand(private val plugin: MedievalFactions) : CommandExecutor {
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
+        if (!sender.hasPermission("mf.help")) {
+            sender.sendMessage("$RED${plugin.language["CommandFactionHelpNoPermission"]}");
+            return true
+        }
+        sender.sendMessage("$AQUA${plugin.language["CommandFactionHelpListTitle"]}");
+
+        // more relevant to members
+        sender.sendMessage("$AQUA${plugin.language["CommandFactionHelpJoin"]}");
```

Comment:
I wish we could do translations for plugin.yml so that /help faction would display help in the user's native tongue. This definitely works in showing users help in their own lqnguage though, and perhaps gives us a little more control over formatting.

Reply:
Yeah that would be nice. This will give is a but more control for sure.